frontend/src/screens/Player/Player.tsx

```diff
@@ -0,0 +1,79 @@
+import Sidebar from "../../components/Sidebar/Sidebar";
+import "./Player.modules.scss";
+import person from "../../assets/icons/person.svg";
+import heart from "../../assets/icons/heart.svg";
+import clock from "../../assets/icons/clock.svg";
```

Comment:
note: With CRA you can import svg as a React component, to render it inline instead of rendering it via the `img` tag:

`import {ReactComponent as ClockIcon} from '../../assets/icons/clock.svg';`